Tests/bootstrap.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Bootstrap file for running the tests on PHPCS 3.x.
+ *
+ * Load the PHPCS autoloader and the WPCS PHPCS cross-version helpers.
+ *
+ * {@internal We need to load the PHPCS autoloader first, so as to allow their
+ * auto-loader to find the classes we want to alias for PHPCS 3.x.
+ * This aliasing has to be done before any of the test classes are loaded by the
+ * PHPCS native unit test suite to prevent fatal errors.
+ *
+ * @package WPCS\WordPressCodingStandards
```

Comment:
I think this docblock still needs adjusting, credit should be given for the file to WPCS, but the package, link and since tags should be for YoastCS.